README.md

```diff
@@ -99,8 +101,10 @@ jobs:
               with:
                 github_token: ${{ secrets.CUSTOM_TOKEN }}
                 files_to_ignore: name_of_file_where_this_action_is_used.yml
+                files_to_include: file.yml,folder/another_file.yml
```

Comment:
maybe make it clear, like a real example with LICENSE or CONTRIBUTION.md file?

Reply:
could you also mention it in the overview at the top of the readme, that it is not just for global workflows but also other files, like community health files, like code of conduct and others. I know workflow could actually do more than copying of these files, but this is a great and clear use case